### ISSUE TITLE
NTR: CT or MRI: spinal extradural lipoma (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8971]
New term request CT or MRI: spinal extradural lipoma (elthat_130302182249):   UMLS: 0347444

**Comment by MickeySegal** [comment]
We appear to be using both senses:

Smith Malformation textbook for Proteus syndrome "spinal lipomatosis"

https://www.omim.org/entry/182940 "The patients had sporadic neural tube defects, sporadic spinal lipoma and tethered cord, and sporadic lumbosacral meningomyelocele, respectively. "

We also have a finding "CT or MRI: spinal intradural lipoma", which we use for CT or MRI: spinal intradural lipoma and other conditions such as https://www.ncbi.nlm.nih.gov/books/NBK576966/

**Comment by pnrobinson** [comment]
@MickeySegal 
note this existing term for the second case:
HP:6000415
Intradural spinal lipoma

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000996
- label: Intraspinal extradural lipoma
- parents: [Spinal cord tumor (HP:0010302)](https://hpo.jax.org/app/browse/term/HP:0010302)
- pmid: PMID:30488010
- def: A lipoma (a benign encapsulated mass with lobulated mature adipose tissue enclosed by fibrous tissue) located within the spine but outside of the dura mater.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8971
- synonyms:
